frontend/tests/controllers/RunDetailsTest.php

```diff
@@ -136,6 +136,34 @@ public function testAdminOutOfContest() {
         );
     }
 
+    public function testDownload() {
+        $adminLogin = self::login($this->admin);
+        $login = self::login($this->identity);
+
+        $runData = \OmegaUp\Test\Factories\Run::createRunToProblem(
+            $this->problemData,
+            $this->identity,
+            $login
+        );
+        \OmegaUp\Test\Factories\Run::gradeRun($runData, 1, 'AC', 50);
+
+        ob_start();
+        try {
+            \OmegaUp\Controllers\Run::apiDownload(new \OmegaUp\Request([
+                'run_alias' => $runData['response']['guid'],
+                'auth_token' => $login->auth_token,
+                'show_diff' => true,
+            ]));
+        } catch (\OmegaUp\Exceptions\ExitException $e) {
+            // Expected exception
+            $zipContents = ob_get_contents();
+            error_log(print_r($zipContents, true));
+        }
```

Comment:
acá se puede usar una técnica similar:

```suggestion
        } catch (\OmegaUp\Exceptions\ExitException $e) {
            // This is expected.
        }
        $zipPath = tmpfile();
        file_put_contents($zipPath, ob_get_contents());
        ob_end_clean();
        
        $zip = new \ZipArchive();
        if ($zip->open($zipPath, \ZipArchive::RDONLY) !== true) {
            throw ...
        }
        // Usar $zip para hacer aseveraciones.
```

Reply:
Listo. Subí las correcciones en el PR #4324